File: Fireblog/DataBaseLinker.php
<?php

class DataBaseLinker{
    private static  $urlBDD="mysql:host=localhost;dbname=fireblog;";
    private static  $username="root";
    private static  $password="root";
    private static  $connexion;
    
    static function getConnexion(){
            self::$connexion = new PDO (self::$urlBDD, self::$username, self::$password);
            return self::$connexion;
    }
}

File: JAVA/SeriesTV/src/main/java/com/sio/seriestv/tools/DataBaseLinker.java
package com.sio.seriestv.tools;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;

public class DataBaseLinker {

    private static String url = "jdbc:mysql://localhost:3306/serieacteur?useSSL=false&serverTimezone=UTC";

    private static String user = "root";
    private static String password = "root";

    private static Connection conn = null;

    public static Connection getConnexion() {
        if (conn == null) {
            try {
                conn = DriverManager.getConnection(url, user, password);
            }
            catch (SQLException e) {
                e.printStackTrace();
            }
        }

        return conn;
    }
}
